survey_merge_coded/survey_merge_coded.py

```diff
@@ -75,6 +74,29 @@ def make_standard_merge_plan(variable_name, survey_name):
             TracedDataCodaIO.import_coda_to_traced_data_iterable(
                 user, surveys, plan_item.key_of_raw, plan_item.scheme_keys, f, True)
 
+    # Items coded under household sickness are not explicitly coded under People. For these cases, set the
+    # people column to 'NC'
+    sickness_people_key = "Household_Sickness (Text) - wt_practice_coded_people"
+    household_sickness_key = "Household_Sickness (Text) - wt_practice_coded"
+    for td in surveys:
+        if td.get(household_sickness_key) is not None and td.get(sickness_people_key) is None:
+            td.append_data({sickness_people_key: "NC"}, Metadata(user, Metadata.get_call_location(), user))
+
+    # Import Trustworthy Advisors using matrix imports
+    coda_file_path = path.join(coded_input_path, "Trustworthy_Advisors_coded.csv")
+    trustworthy_advisors_raw_key = "Trustworthy_Advisors (Text) - wt_practice"
+
+    with open(coda_file_path, "r") as f:
+        TracedDataCodaIO.import_coda_to_traced_data_iterable_as_matrix(
+            user, surveys, trustworthy_advisors_raw_key, {"Trustworthy advisors", "Trustworthy advisors 2"}, f,
+            key_of_coded_prefix="{}_coded_".format(trustworthy_advisors_raw_key))
+
+    with open(coda_file_path, "r") as f:
+        TracedDataCodaIO.import_coda_to_traced_data_iterable_as_matrix(
+            user, surveys, trustworthy_advisors_raw_key,
+            {"what to do during an ourbreak", "what to do during outbreak"}, f,
```

Comment:
It's hard reading this code to understand what this set is being used for. Consider assigning to a variable to explain its purpose.